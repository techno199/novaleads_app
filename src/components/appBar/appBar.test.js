import React from 'react'
import renderer from 'react-test-renderer'
import { AppBar } from '.';

test('should render application top bar', () => {
  const component = renderer.create(
    <AppBar title='title' />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
