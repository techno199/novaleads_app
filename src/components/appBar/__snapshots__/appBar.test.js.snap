// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render application top bar 1`] = `
<header
  className="MuiPaper-root-10 MuiPaper-elevation4-16 MuiAppBar-root-1 MuiAppBar-positionFixed-2 MuiAppBar-colorPrimary-8 mui-fixed"
>
  <div
    className="MuiToolbar-root-37 MuiToolbar-regular-39 MuiToolbar-gutters-38"
  >
    <h6
      className="MuiTypography-root-41 MuiTypography-h6-58 MuiTypography-colorInherit-70"
    >
      title
    </h6>
  </div>
</header>
`;
